packages/web-config-server/examples.http

```diff
@@ -1,29 +1,39 @@
-@hostname = localhost
-@port = 8000
-@host = {{hostname}}:{{port}}
-@version = v1
+@localhost = http://localhost:8000
```

Comment:
Some changes in here that allow it to be easier to swap between local and dev for making comparisons (as well as making some things harder to improve simplicity, like swapping api version, which we never do)